pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -6,7 +6,9 @@
 , binaryName ? "firefox"
 , application ? "browser"
 , applicationName ? "Mozilla Firefox"
+, appstreamMetadataSourceFile ? "../taskcluster/docker/${pname}-flatpak/${distributionId}.appdata.xml.in"
 , branding ? null
+, distributionId ? "org.mozilla.firefox"
```

Comment:
Should this possibly also be used for setting `--with-distribution-id` in `configureFlags` too? Or is it intentional that the baked-in and appstream distribution-id differ?